cms/static/sass/course-unit-mfe-iframe-bundle.scss

```diff
@@ -0,0 +1,645 @@
+      content: '';
+      position: absolute;
+      background-color: $black;
+      mask: url('#{$static-path}/images/pencil-icon.svg') center no-repeat;
+      top: 11px;
+      left: 11px;
+      width: 20px;
+      height: 20px;
+    }
+
+    &:hover {
+      background-color: $primary;
+      border-color: $transparent;
+
+      &::before {
+        background-color: $white;
+      }
+    }
+
+    &:focus {
+      outline: 2px $transparent;
+      background-color: $transparent;
+      box-shadow: inset 0 0 0 2px $primary;
+      border-color: $transparent;
+
+      &:hover {
+        color: $white;
+        border-color: $transparent;
+
+        &::before {
+          background-color: $primary;
+        }
+      }
+    }
+  }
+}
+
+.nav-dd.ui-right .nav-sub {
+  &::before,
+  &::after {
+    display: none;
+  }
+}
+
+[class*="view-"] .modal-lg.modal-editor .modal-header .editor-modes .action-item .editor-button,
+[class*="view-"] .modal-lg.modal-editor .modal-header .editor-modes .action-item .settings-button,
+#openassessment-editor #openassessment_editor_header .editor_tabs .oa_editor_tab {
+  color: $black;
+  background-color: $light-background-color;
+  border-color: $light-background-color;
+  box-shadow: none;
+  border: 1px solid $transparent;
+  font-weight: 500;
+
+  @extend %modal-actions-button;
+}
+
+[class*="view-"] .modal-window .modal-actions .action-primary,
+body .xblock--drag-and-drop--editor .btn,
+body #openassessment-editor .openassessment_container_add_button,
+body #openassessment-editor #oa_rubric_editor_wrapper .openassessment_criterion .openassessment_criterion_add_option {
+  background-color: $primary;
+  color: $white;
+  border: 1px solid $primary;
+  cursor: pointer;
+  background-image: none;
+  display: block;
+
+  @extend %modal-actions-button;
+
+  &:hover {
+    background: darken($primary, 5%);
+    border-color: #2d494e;
+    box-shadow: none;
+  }
+
+  &:focus {
+    background: $primary;
+    outline: none;
+    box-shadow: none;
+  }
+}
+
+body .xblock--drag-and-drop--editor .btn {
+  display: inline-block;
+}
+
+[class*="view-"] .modal-window .modal-actions a {
+  color: $text-color;
+  background-color: $transparent;
+  border-color: $transparent;
+  box-shadow: none;
+  font-weight: 500;
+  border: 1px solid $transparent !important;
+  padding: 10px 16px !important;
+  font-size: $base-font-size;
+  line-height: 20px;
+  border-radius: $btn-border-radius !important;
+  text-align: center;
+  vertical-align: middle;
+  user-select: none;
+  background-image: none;
+  display: block;
+
+  &:hover {
+    background-color: $light-background-color;
+    border-color: $transparent;
+    color: $text-color;
+  }
+
+  &:focus {
+    background: $transparent;
+    outline: none;
+  }
+}
+
+body .nav-dd .nav-sub {
+  border: 1px solid rgba(0, 0, 0, .15);
+  border-radius: 6px;
+  padding: 10px 0;
+  min-width: 288px;
+  right: 90px;
+
+  .nav-item {
+    border-bottom: none;
+    color: #212529;
```

Comment:
[question] Can we use existing variables or create new variables for colors instead of hardcoding them?

Reply:
Created some new variables